parlai/tasks/task_list.py

```diff
@@ -383,4 +383,11 @@
         "description": "Dump of Wikipedia articles from 2/3/18",
         "notes": "Specify ':full' for the full articles to be returned, otherwise defaults to ':summary', which provides the first paragraphs. To put the article in the labels and the title in the text, specify ':key-value' at the end (for a title/content key-value association)"
     },
+    {
+        "id": "Reddit",
+        "display_name": "Reddit",
+        "task": 'reddit',
+        "tags": [ "All", "ChitChat" ],
+        "description": "Subreddit data"
+    }
```

Comment:
oops

Reply:
Failed to add this part. It's an ongoing work, not ready to commit yet. Maybe delete these lines for now?

Reply:
yes please delete and get in the other fix

Reply:
OK, no problem